File: practice/algorithms/search/missing_numbers/missing_numbers.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
    int n, m, v;
    
    cin >> n;
    vector<int> seqA(10000);
    for (int pos = 0; pos < n; ++pos)
    {
        cin >> v;
        seqA[v] += 1;
    }
    
    cin >> m;
    vector<int> seqB(10000);
    for (int pos = 0; pos < m; ++pos)
    {
        cin >> v;
        seqB[v] += 1;
    }
    
    for (int pos = 0; pos < 10000; ++pos)
    {
        if (seqB[pos] > seqA[pos])
        {
            printf("%d ", pos);
        }
    }
    
    return 0;
}

File: practice/algorithms/search/missing_numbers/missing_numbers.py
#!/usr/bin/env python

n = int(raw_input().strip())
a = map(int, raw_input().strip().split())
seqA = [0 for _ in range(10000)]
for VAL in a:
    seqA[VAL] += 1
    
m = int(raw_input().strip())
b = map(int, raw_input().strip().split())
seqB = [0 for _ in range(10000)]
for VAL in b:
    seqB[VAL] += 1

for POS in range(10000):
    if seqB[POS] > seqA[POS]:
        print POS,

